src/utils/UserUtils.tsx

```diff
@@ -0,0 +1,55 @@
+import { FIRST_ALPHABET_REGEX, SPLITTING_WORDS_REGEX } from '../@config/constants'
+
+/**
+ * This is a helper function to help the wallet user see prefilled initials.
+ *
+ * @param namePart1 is first part of the username. Also known as firstName
+ * @param namePart2 is 2nd part of the username. Also known as lastName, surname or remaining part of the name etc.
+ *
+ * @return the initials selected by default
+ */
+export const getDefaultUserNameInitials = (namePart1?: string, namePart2?: string): string => {
+  let initials = ''
+  initials += getPartialInitials(namePart1)
+  initials += getFirstAlphabetOfLastWord(namePart2)
+  if (initials.length < 1) {
+    initials = '?'
+  }
+
+  return initials.toUpperCase()
+}
+
+/**
+ * This is a helper function to help the wallet user see prefilled initials.
+ *
+ * @param namePart is part of the username
+ *
+ * @return the partial initials selected by default
+ */
+export const getPartialInitials = (namePart?: string): string => {
```

Comment:
this feels like a lot of code just to get 2 letters out of a full name

Reply:
Removed the complex code. Made it simple as suggested by @Brummos